sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_multimodal/_content_safety_multimodal.py

```diff
@@ -0,0 +1,232 @@
+    from ._sexual import SexualMultimodalEvaluator
+    from ._violence import ViolenceMultimodalEvaluator
+except ImportError:
+    from _hate_unfairness import HateUnfairnessMultimodalEvaluator
+    from _self_harm import SelfHarmMultimodalEvaluator
+    from _sexual import SexualMultimodalEvaluator
+    from _violence import ViolenceMultimodalEvaluator
+
+logger = logging.getLogger(__name__)
+
+class ContentSafetyMultimodalEvaluator:
+    """
+    Initialize a content safety multimodal evaluator configured to evaluate content safety metrics for multimodal scenario.
+    :param azure_ai_project: The scope of the Azure AI project.
+        It contains subscription id, resource group, and project name.
+    :type azure_ai_project: ~azure.ai.evaluation.AzureAIProject
+    :param parallel: If True, use parallel execution for evaluators. Else, use sequential execution.
+        Default is True.
+    :type parallel: bool
+    :param credential: The credential for connecting to Azure AI project.
+    :type credential: ~azure.core.credentials.TokenCredential
+    :return: A function that evaluates multimodal chat messages and generates metrics.
+    :rtype: Callable
+    
+    **Usage**
+    .. code-block:: python
+        azure_ai_project = {
+            "subscription_id": "<subscription_id>",
+            "resource_group_name": "<resource_group_name>",
+            "project_name": "<project_name>",
+        }
+        eval_fn = ContentSafetyMultimodalEvaluator(azure_ai_project)
+        result = eval_fn(
+            messages= [
+                {
+                    "role": "user",
+                    "content": [
+                        {
+                            "type": "text",
+                            "text": "What's in this image?"
+                        },
+                        {
+                            "type": "image_url",
+                            "image_url": {
+                                "url": "<image url or base64 encoded image>"
+                            }
+                        }
+                    ]
+                },
+                {
+                    "role": "assistant",
+                    "content": "This picture shows an astronaut standing in desert."
+                }
+            ]
+        )
+        
+    **Output format**
+    .. code-block:: python
+        {
+            "violence": "Medium",
+            "violence_score": 5.0,
+            "violence_reason": "Some reason",
+            "sexual": "Medium",
+            "sexual_score": 5.0,
+            "sexual_reason": "Some reason",
+            "self_harm": "Medium",
+            "self_harm_score": 5.0,
+            "self_harm_reason": "Some reason",
+            "hate_unfairness": "Medium",
+            "hate_unfairness_score": 5.0,
+            "hate_unfairness_reason": "Some reason"
+        }
+    """
+    
+    def __init__(self, azure_ai_project: dict, parallel: bool = False, credential=None):
```

Comment:
```suggestion
    def __init__(self, credential, azure_ai_project, parallel: bool = False):
```

(applies to all new evaluator `__init__`)